apps/dash-mapd-demo/app.py

```diff
@@ -0,0 +1,649 @@
+import dash
+import dash_table
+import dash_core_components as dcc
+import dash_html_components as html
+import plotly.graph_objs as go
+
+import numpy as np
+import pandas as pd
+from dash.dependencies import State, Input, Output
+from dash.exceptions import PreventUpdate
+import pymapd
+import datetime
+from datetime import datetime as dt
+import os
+
+app = dash.Dash(__name__)
+server = app.server
+
+app.config.suppress_callback_exceptions = True
+
+# Get credentials from env
+user = "mapd"
+password = "HyperInteractive"
```

Comment:
How secure do these credetials have to be? Are they just dummy credentials? Also, it doesn't look like they're being  read from environment variables (since they're hardcoded).

Reply:
this is public login credentials for open-source version db(common for all CE version so doesn't need to be protected).  @tarzzz installed it externally on google cloud instances with restricted connectivity to only be allowed from the dash gallery instance (https://github.com/plotly/streambed/issues/13013).  The only credentials should not be exposed is host URL in this way.